lib/tasks/taxonomy/analysis/taxons_tagged_to_content.rake

```diff
@@ -3,7 +3,7 @@ require 'csv'
 
 namespace :taxonomy do
   namespace :analysis do
-    desc "Produces a CSV list of content items under a root taxon ID, and the taxons they are tagged to"
+    desc "Produces a CSV list of links under a root taxon ID, and the base paths they are tagged to"
```

Comment:
I think removing the word "taxons" from this description is a bit confusing - possibly "...the taxon base paths they are..." ?